Adyen.xcodeproj/project.pbxproj

```diff
@@ -1768,30 +1793,31 @@
 		E278EB3C22AA5E8700497FD5 /* Payment Methods */ = {
 			isa = PBXGroup;
 			children = (
-				E788A4EE2658030100089448 /* DisplayInformation.swift */,
-				E9B36C9C223FDE4F00EAA368 /* PaymentMethods.swift */,
-				F9D5751723799BB0009C18B5 /* AnyPaymentMethodDecoder.swift */,
-				E262128E22A90D8800F2B133 /* PaymentMethod.swift */,
+				C982FFDB2694792F00AED849 /* AffirmPaymentMethod.swift */,
 				F9D5753023828EBB009C18B5 /* AnyCardPaymentMethod.swift */,
-				E278EB4022AA5EBD00497FD5 /* CardPaymentMethod.swift */,
+				F9D5751723799BB0009C18B5 /* AnyPaymentMethodDecoder.swift */,
+				E95E24BF22C2593900973184 /* ApplePayPaymentMethod.swift */,
 				F9D5751B237C0EA0009C18B5 /* BCMCPaymentMethod.swift */,
-				F9D57529237C6084009C18B5 /* StoredBCMCPaymentMethod.swift */,
+				E700A72C2549E2F00012B64C /* BLIKPaymentMethod.swift */,
+				5A988BF5265675ED0007F4C0 /* BoletoPaymentMethod.swift */,
+				E278EB4022AA5EBD00497FD5 /* CardPaymentMethod.swift */,
+				F9FE25B32625ECE7001874BB /* CustomDisplayablePaymentMethod.swift */,
+				E788A4EE2658030100089448 /* DisplayInformation.swift */,
+				F97C837825BF047600D7F85C /* DokuPaymentMethod.swift */,
+				E788A3B1265565DF00089448 /* EContextPaymentMethod.swift */,
+				E7763A1025F02BD20046371E /* GiftCardPaymentMethod.swift */,
 				E278EB4222AA5EC800497FD5 /* IssuerListPaymentMethod.swift */,
-				E278EB4422AA5ED600497FD5 /* SEPADirectDebitPaymentMethod.swift */,
-				E224088C22B0FD220058923E /* StoredPayPalPaymentMethod.swift */,
+				F9D644DD24D2D9950059CBE3 /* MBWayPaymentMethod.swift */,
+				E262128E22A90D8800F2B133 /* PaymentMethod.swift */,
+				E9B36C9C223FDE4F00EAA368 /* PaymentMethods.swift */,
+				F926D52B23F59ABE00D058D3 /* QiwiWalletPaymentMethod.swift */,
 				E224088622AFA3A80058923E /* RedirectPaymentMethod.swift */,
+				E278EB4422AA5ED600497FD5 /* SEPADirectDebitPaymentMethod.swift */,
 				E27BFD8F22B114DC00692E0D /* ShopperInteraction.swift */,
-				E95E24BF22C2593900973184 /* ApplePayPaymentMethod.swift */,
-				F9620D9E23C7457F005209FC /* WeChatPayPaymentMethod.swift */,
-				F926D52B23F59ABE00D058D3 /* QiwiWalletPaymentMethod.swift */,
-				F9D644DD24D2D9950059CBE3 /* MBWayPaymentMethod.swift */,
-				F97C837825BF047600D7F85C /* DokuPaymentMethod.swift */,
-				E700A72C2549E2F00012B64C /* BLIKPaymentMethod.swift */,
-				E7763A1025F02BD20046371E /* GiftCardPaymentMethod.swift */,
-				F9FE25B32625ECE7001874BB /* CustomDisplayablePaymentMethod.swift */,
+				F9D57529237C6084009C18B5 /* StoredBCMCPaymentMethod.swift */,
 				E71E8F3D257921D10054B03D /* StoredBLIKPaymentMethod.swift */,
-				E788A3B1265565DF00089448 /* EContextPaymentMethod.swift */,
-				5A988BF5265675ED0007F4C0 /* BoletoPaymentMethod.swift */,
+				E224088C22B0FD220058923E /* StoredPayPalPaymentMethod.swift */,
+				F9620D9E23C7457F005209FC /* WeChatPayPaymentMethod.swift */,
 			);
```

Comment:
Did you ordered payment methods alphabetically? :D

Reply:
I did. With so many of them it was getting difficult to find Affirm sometimes. Anything wrong?

Reply:
No! Well done :D
I was just curious why so many changes :)